presentation/src/main/java/com/android254/presentation/home/components/HomeSessionSection.kt

```diff
@@ -112,7 +117,8 @@ fun HomeSectionHeader(
                 text = stringResource(id = R.string.view_all_label),
                 textAlign = TextAlign.Start,
                 style = TextStyle(
-                    color = ChaiBlue,
+                    //made the color for the sessions dynamic for Dark mode visibility
+                    color = if (isSystemInDarkTheme()) Color.White else ChaiBlue,
```

Comment:
This is already handled in the `ChaiDCKE22Theme`. Check out `ChaiColors`, You can add an entry on  `ChaiLightColorPalette` and `ChaiDarkColorPalette` and use it in code like this `MaterialTheme.chaiColorsPalette.your-color`

Reply:
@githukueliud check this PR. https://github.com/droidconKE/droidconKeKotlin/pull/212 should give you an idea of what I mean.

Reply:
ok, on it